crates/debug/src/transform/address_transform.rs

```diff
@@ -50,13 +49,16 @@ struct Range {
     positions: Box<[Position]>,
 }
 
+type RangeIndex = usize;
```

Comment:
I wonder if we could make this a `u32` in practice? Do we ever have to process Wasm files larger than 4GiB?

This would let us shave off half the overhead on 64-bit platforms.